### ISSUE TITLE
Quotes closing issue with script

### ISSUE THREAD

**Issue by Zaky7** [nicolasfara/conventional-commits#80]
Hi @nicolasfara 

Can check if the script is properly closed

https://github.com/nicolasfara/conventional-commits/blob/master/src/main/kotlin/it/nicolasfarabegoli/gradle/ConventionalCommitScript.kt#L42

```
        conventional_commits_regex="^($typesRegex)(\\($scopesRegex))?!?: .+$"
```

it is not working as expected

**Comment by nicolasfara** [comment]
Hi @Zaky7, thanks for reporting the issue!

It's not clear to me what you mean by "is not properly closed"?
If you have configured the plugin in the `build.gradle.kts`, could you show the configuration and the resulting script in `.git/hooks/commit-msg`?

Thanks :)

**Comment by Zaky7** [comment]
@nicolasfara yeah next time I will be more verbose. In which version it will be fixed?

**Comment by nicolasfara** [comment]
It's fixed in the `3.0.3`, which is just been released!
